src/libs/relay/conduit_relay_io_hdf5.cpp

```diff
@@ -682,11 +724,17 @@ conduit_dtype_to_hdf5_dtype_cleanup(hid_t hdf5_dtype_id,
 //-----------------------------------------------------------------------------
 DataType
 hdf5_dtype_to_conduit_dtype(hid_t hdf5_dtype_id,
-                            index_t num_elems,
+                            hsize_t * num_elems_array,
```

Comment:
can we keep the old signature and have it plumbed into this new general one?

Reply:
Good idea.

Reply:
Done.